README.md

```diff
@@ -37,3 +37,6 @@ Axios is a promise-based HTTP client for the browser and Node.js. It provides an
 ### Tailwind CSS
 
 Tailwind CSS is a utility-first CSS framework for rapidly building custom user interfaces. It provides a set of pre-defined utility classes that you can use to style your components.
+
+
+### This is Katie testing this out! First PR!
```

Comment:
This is great @hellok8ie! If you wanted to add hyperlinks to all the resources in the read me, we could approve this an add it to main. Google or chat GPT how to add hyperlinks in markdown. Then you can use GitHub to preview what it will end up looking like. 
![image](https://github.com/Firm-Collective/One-Accord/assets/77178810/9362af4f-3d7d-4923-a3d9-d080c9b477db)